ProjectSidewalk/SidewalkWebpage
# user_id constraint added to webpage activity table

Resolves #2091 

Added a foreign key constraint to the user_id column in webpage activity. Had to adjust temporary_label_id in a couple format files from nullable to non-null. 

##### Testing instructions
1. Use preferred choice of database management tool
2. Check webpage_activity table for any rows with a user_id being NULL or a user_id not in the sidewalk_user table. Should be 0

##### Things to check before submitting the PR
